my-page-app/src/components/admin/Admin.tsx

```diff
@@ -15,6 +16,8 @@ function Admin() {
   const [expandedUser, setExpandedUser] = useState<string>('')
   const [filterValue, setFilterValue] = useState('')
 
+  const [activeBudget, setActiveBudget] = useState(null as AccordionEventKey | null)
```

Comment:
```suggestion
  const [activeBudget, setActiveBudget] = useState<AccordionEventKey | null>(null)
```

For eksempel. Casting er litt uheldig i dette tilfellet.

Trenger dette å ligge i Admin? Kan det ikke leve i BudgetList komponenten i stedet? Tenker at Admin komponenten ikke har noe med hvilket budsjett som er åpent eller ikke. Vil få samme effekten ved å definere staten i BudgetList 😄